### DISCUSSION TITLE
Single project

### DISCUSSION THREAD

**Discussion by Bo0m21** [dotnet/maui#2309] [Q&A] [answered]
Hello everyone! Will the next versions combine everything into a single project after creating a new project?
Now two projects are being created, the second one with the name WinUI at the end.
Why were they divided by 2 and not by 4, for example?

**Comment by saint4eva** [comment, marked as answer]
> Hello everyone! Will the next versions combine everything into a single project after creating a new project?
> Now two projects are being created, the second one with the name WinUI at the end.
> Why were they divided by 2 and not by 4, for example?

The project is still a work in progress. The final aim is to combine all the project heads into a single project. The WinUI team is still working things out. Once they are through with it, the Maui team will combine it to a single project .

**Reply by Bo0m21** [reply]
Thanks, wait release :)
